### DISCUSSION TITLE
Japanese Translation

### DISCUSSION THREAD

**Discussion by suzukiyuuma** [dbeaver/dbeaver#11725] [DBeaver Development] [answered]
I would like to contribute Japanese translation.
Could you see it if I fork this repo and send pull-request?
Or could I join the team like korean or chinese persons?

**Comment by ShadelessFox** [comment, marked as answer]
Hello @suzukiyuuma,

You can follow [this article](https://github.com/dbeaver/dbeaver/wiki/Localization) on how to add translation, it will require to fork this repo and make a pull request, yeah :)

Thanks for future contributions!

P.S: I will convert this issue to a discussion.
